catroid/src/org/catrobat/catroid/content/bricks/WhenNfcBrick.java

```diff
@@ -80,6 +80,10 @@ public WhenNfcBrick(WhenNfcScript script) {
 		this.oldSelectedNfcTag = null;
 		this.nfcTag = script.getNfcTag();
 		this.whenNfcScript = script;
+
+		if (script != null && script.isCommentedOut()) {
+			setCommentedOut(true);
+		}
```

Comment:
Agreed, it can always be copied, that's no problem. But still I'm not sure how this can be achieved with the least effort. 

in a class SpecialBrick, we override the method
Brick clone() 
which returns a SpecialBrick that is derived from BrickBasetype.
If i want to do the copying of the variable, I'd have to create the following member in BrickBaseType:
Brick copy = null; 
and the clone() method in BrickBaseType looks as follows:
copy.isCommentedOut = isCommentedOut

and in the  SpecialBrick clone() method looks as follows:
{
    copy = new SpecialBrick();
    // or: copyBaseValues()
    return super.clone(); // now we can copy the value (and maybe other values in the future)
} 

1) I'm not sure if that's a good design
2) I still have to modify every Brick to call a method from BrickBaseType. That would only pay off if multiple values have to be copied (maybe in future)

Or how would you suggest to solve this? Thanks for any hint! :+1: